python/lsst/meas/base/applyApCorr.py

```diff
@@ -0,0 +1,159 @@
+#
+# LSST Data Management System
+# Copyright 2015 LSST Corporation.
+#
+# This product includes software developed by the
+# LSST Project (http://www.lsst.org/).
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the LSST License Statement and
+# the GNU General Public License along with this program.  If not,
+# see <http://www.lsstcorp.org/LegalNotices/>.
+#
+import math
+
+import lsst.pex.config
+import lsst.pex.exceptions
+import lsst.afw.image
+import lsst.pipe.base
+from .base import getApCorrNameSet
+
+__all__ = ("ApplyApCorrConfig", "ApplyApCorrTask")
+
+class ApCorrInfo(object):
+    """!Catalog field names and keys needed to aperture correct a particular flux
+    """
+    def __init__(self, schema, algName):
+        """!Construct an ApCorrInfo and add fields to the schema
+
+        @param[in,out] schema  source catalog schema;
+            three fields are used to generate keys:
+            - _algName_\_flux
+            - _algName_\_fluxSigma
+            - _algName_\_fluxFlag
+            three fields are added:
+            - _algName_\_ApCorr
+            - _algName_\_ApCorrSigma
+            - _algName_\_flag_apcorr
+        @param[in] algName  name of flux measurement algorithm, e.g. "base_PsfFlux"
+        """
```

Comment:
I find your format string syntax here a little unusual; does this get translated into boldface in HTML or something?

If that's all you're going for, I'd recommend using curly braces or something to set off the variable name (i.e. use Python `format()` syntax).

In general, I think if Doxygen markup interpretation requires us to do this much damage to the raw text form of a docstring (which is still visible in interactive help, after all), it's best to just disable the Doxygen markup by not including the initial `!`.